### PR TITLE
add custom events when prns fetched

### PR DESCRIPTION
**Pull request by Ravi-SharmaGautam**
Added changes to add customevents when npwd are fetched

### CODE REVIEW

File: src/EprPrnIntegration.Common.UnitTests/Helpers/UtilitiesTests.cs [hunk 1]
```diff
@@ -12,12 +15,23 @@ public class UtilitiesTests
     private readonly Mock<IServiceBusProvider> _serviceBusProviderMock;
     private readonly Mock<IConfiguration> _configurationMock;
     private readonly Utilities _utilities;
+    private readonly Mock<ITelemetryChannel> _telemetryChannel;
```
Comment (stephan-williamson) [inline src/EprPrnIntegration.Common.UnitTests/Helpers/UtilitiesTests.cs:18]: Please name as mock e.g. _telemetryChannelMock

File: src/EprPrnIntegration.Common/Helpers/Utilities.cs [hunk 2]
```diff
@@ -23,4 +24,9 @@ public async Task SetDeltaSyncExecution(DeltaSyncExecution syncExecution, DateTi
         syncExecution.LastSyncDateTime = latestRun;
         await serviceBusProvider.SendDeltaSyncExecutionToQueue(syncExecution);
     }
+
```
Comment (stephan-williamson) [inline src/EprPrnIntegration.Common/Helpers/Utilities.cs:27]: This Utilities class looks like it might become a repository of miscellaneous unrelated stuff

Comment (Ravi-SharmaGautam) [reply src/EprPrnIntegration.Common/Helpers/Utilities.cs:27]: I have moved telemetry in utilities as function has already too many injections. Considering utilities can be used for misc things. I will leave it there for now. We can refactor later if required

File: src/EprPrnIntegration.Common.UnitTests/Helpers/UtilitiesTests.cs (hunk 1 above)
Comment (Ravi-SharmaGautam) [reply src/EprPrnIntegration.Common.UnitTests/Helpers/UtilitiesTests.cs]: changed

**Review by stephan-williamson — APPROVED** [review]

**Review by Ehsan-Hatami — APPROVED** [review]
